File: Python/Iniciante/1020.py
# Problema 1020 - Beecrowd - Iniciante - Nível 2

# Vamos ler o número de dias e converter o valor para inteiro
dias = int(input())

# Começamos calculando o número de anos
anos = dias//365
dias = dias%365

# Depois o número de meses
meses = dias//30
dias  = dias%30

# Então printamos todos os valores
print(f'{anos} ano(s)')
print(f'{meses} mes(es)')
print(f'{dias} dia(s)')

File: Rust/Iniciante/1020.rs
use std::io;
use std::error::Error;

fn main() -> Result<(), Box<dyn Error>> {
	let stdin = io::stdin();
	let mut buffer = String::new();

	stdin.read_line(&mut buffer)?;
	let mut dias: i64 = buffer.trim().parse()?;

	let anos: i64 = dias / 365;
	dias %= 365;

	let meses: i64 = dias / 30;
	dias %= 30;

	println!("{} ano(s)", anos);
	println!("{} mes(es)", meses);
	println!("{} dia(s)", dias);

	Ok(())
}